src/bindings/js/node/tests/model.test.js

```diff
@@ -111,4 +111,44 @@ describe('Model.getOutputSize()', () => {
   it('should return 1 for the default model', () => {
     assert.strictEqual(model.getOutputSize(), 1, 'Expected getOutputSize to return 1 for the default model');
   });
+});
+
+describe('Model.getOutputElementType()', () => {
+  it('should return a string for the element type ', () => {
+    const result = model.getOutputElementType(0);
+    assert.strictEqual(typeof result, 'string', 
+      'getOutputElementType() should return a string');
+  });
+
+  it('should accept a single number argument', () => {
+    assert.throws(() => {
+      model.getOutputElementType();
+    }, /^Error: Invalid argument. Expected a single number for output index\.$/,
```

Comment:
After you changed the method of parameter validation this tests will fail.
Update error messages

Reply:
Have updated the error methods to follow strings defined in type validation.